styles/style.css

```diff
@@ -0,0 +1,159 @@
+
+html, body{height:100%;}
+
+body{
+    background-image: url(images/background_image.jpg) ;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-color: black;
+    color: white;
+    -webkit-padding-start: 5em;
+    -webkit-padding-end: 5em;  
+    font-family:  'Futura', 'Century Gothic','Tahoma', sans-serif;
+    font-weight: 100; 
+}
+.container-header{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-wrap: wrap;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+    
+}
+
+.btn {
+    background: #2c2c2c;
+    background-image: -webkit-linear-gradient(#4c4c4c 0%, #2c2c2c 0%, #595959 0%, #131313 20%, #131313 100%);
+    background-image: -moz-linear-gradient(#4c4c4c 0%, #2c2c2c 0%, #595959 0%, #131313 20%, #131313 100%);
+    background-image: -o-linear-gradient(#4c4c4c 0%, #2c2c2c 0%, #595959 0%, #131313 20%, #131313 100%);
+    background-image: linear-gradient(#4c4c4c 0%, #2c2c2c 0%, #595959 0%, #131313 20%, #131313 100%);
+    -webkit-box-shadow: -1px -1px 2px white;
+    -moz-box-shadow: -1px -1px 2px white;
+     box-shadow: -1px -1px 2px white;
+     font-size: 16px;
+     padding: 10px 20px 10px 20px;
+     text-decoration: none;
+     font-family: 'Franklin Gothic Medium', 'Arial Narrow', sans-serif;
+     color: white;
+  }
+
+button.nav-item.active.btn{
+  flex: 1;
+}
+
+button.nav-item-1.btn{
+  flex: 1;
+}
+button.nav-item-2.btn{
+  flex: 1;
+}
+button.nav-item-3.btn{
```

Comment:
You've got 3 CSS classes here that do the same thing. Perhaps you could just reuse one of them?